internal/server/statvar/golden/get_statvar_match/energy_in_us_nomodel.json

```diff
@@ -0,0 +1,64 @@
+{
+  "matchInfo": [
+    {
+      "statVar": "Annual_Exports_Energy_Heat",
+      "statVarName": "Annual Exports of Energy: Heat",
+      "score": 2.8659846941335667,
+      "explanation": "sn:\"energy\" sn:\"in\" sn:\"us\" \n2.87 =\u003e product of:\n 2.87 =\u003e sum of:\n  2.87 =\u003e sum of:\n   2.87 =\u003e weight(sn:energy^1.000000 in \u0000\u0000\u0000\u0000\u0000\u0000q), product of:\n    1.00 =\u003e queryWeight(sn:energy^1.000000), product of:\n     1.00 =\u003e boost\n     4.96 =\u003e idf(docFreq=1444, maxDocs=76107)\n     0.20 =\u003e queryNorm\n    2.87 =\u003e fieldWeight(sn:energy in \u0000\u0000\u0000\u0000\u0000\u0000q), product of:\n     1.41 =\u003e tf(termFreq(sn:energy)=2\n     0.41 =\u003e fieldNorm(field=sn, doc=\u0000\u0000\u0000\u0000\u0000\u0000q)\n     4.96 =\u003e idf(docFreq=1444, maxDocs=76107)\n 1.00 =\u003e coord(1/1)\n"
```

Comment:
Is it possible not to have unicode here?

Reply:
Not really. Better, not without changing the bleve library. So I'll just hide the explanation in the test.